### DISCUSSION TITLE
创意工坊密钥如何分享？

### DISCUSSION THREAD

**Discussion by CSSZYF** [pjy612/SteamManifestCache#375] [Question / Answer]
发现就算是有本体游戏账号能正常下载创意工坊 切换到其他账号也没用 
看Pro更新了关于创意工坊的信息 但是不知道如何提取分享创意工坊的密钥

**Comment by pjy612** [comment]
> 发现就算是有本体游戏账号能正常下载创意工坊 切换到其他账号也没用

这个看游戏的，最好是配合第三方网站确认工坊是否限制匿名
如果不限制的话，随便下载。限制的话，自己的账号先订阅mod，然后切到有游戏的号，等它下载完mod再切回来。
目前测了下 发现换号 不影响工坊在有权限的号上下载。

> 看Pro更新了关于创意工坊的信息 但是不知道如何提取分享创意工坊的密钥

工坊的密钥 和 其他密钥在一起 不过 depotid = appid

**Reply by CSSZYF** [reply]
感谢大佬回复，我验证的游戏的是觅长生，起初我在db没看到清单，还以为是清单老旧，找了离线号验证发现是创意工坊加密就过来问了，大佬的办法会遇到“小红车”问题，离线号会下载一堆没用的mod（又不是不能玩）
不过我没理解“depotid = appid”游戏的appid我倒是会找 但是创意工坊的不知道是哪个一个

**Reply by pjy612** [reply]
> 工坊的密钥 和 其他密钥在一起 不过 depotid = appid

这个还能怎么解释？？？
如果你发现 appid 也有 清单 和 密钥的话。
那那个密钥就是工坊的密钥。。。

**Reply by CSSZYF** [reply]
啊 理解大佬说的了 也就是原本的游戏appid 例如游戏id1189 创意工坊的也是1189 不过好像并不能简单的给一个密钥和预估清单就行了 每个mod好像都要清单 是这样的吗？

**Reply by pjy612** [reply]
是的，如果游戏是 1189 并且它有工坊，那么工坊的密钥 在配置文件里面 就对应的 1189。
然后工坊所有mod的清单共用相同的密钥。
但是 mod 对应的清单都不同。 清单id 一般是 工坊mod 链接上的id。
所以要尝试用工坊 一般是 需要找个有游戏的号 配合来下mod的。
极个别游戏 对 工坊下载 无限制，可以在有密钥的情况下随便下载，比如 潜渊症 啥的。

**Reply by CSSZYF** [reply]
完全理解 要是能配置账号到后台自动下载清单密钥就好了

**Reply by pjy612** [reply]
目前没有这种东西，第三方工坊下载网站上不是有教，怎么用 steamcmd 下载工坊内容么？
你可以不切好直接用 steamcmd 登陆有游戏的号，下载工坊内容。 
然后 自己的号上订阅刷新就行了。
但是前提是得有号

**Reply by CSSZYF** [reply]
是的 并且学会了 很有用的操作 感谢大佬 之前找了半天的第三方创意工坊下载都没找到 在大佬这里找到打开还发现还收藏过 吐了
（edge的搜索真的是烂）
然后想问一句不相干的 突然有部分游戏打开失败 报错54之类的 用Steamless处理之后倒是正常能打开了 但有两个问题 一个就是潜水员戴夫是报错弹窗 第二种的
![image](https://github.com/pjy612/SteamManifestCache/assets/63802741/984a8a69-e317-4cc1-b428-fc466fab5bcd)
然后也没办法处理 会失败报错 第二个问题就是这些原本都是正常能打开的 所以是不是tools关了什么功能 有其他代替吗？
我看tg群里说是关掉了授权验证 是这个原因吗？

**Reply by CSSZYF** [reply]
是解锁拉闸了 倒卖真恶心 目前还没有看到新的倒卖大佬的 大佬挺住 有倒卖您的我第一时间举报

**Reply by pjy612** [reply]
我的现在还能卖的起来的话 那绝对是有水平的
